<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>otvet</class>
 <widget class="QDialog" name="otvet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Удачи!</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bg_image/images/Фон_1.PNG);</string>
  </property>
  <widget class="QPushButton" name="Next">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>530</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Продолжить</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);
font: 16pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:18pt;&quot;&gt;dsf&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="otvet1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);
font: 10pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="otvet2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="otvet3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);
selection-background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="otvet4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="area">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>591</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bg_image/images/White);

</string>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Vopros">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>dzssa</string>
   </property>
  </widget>
  <action name="pod">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F10</string>
   </property>
  </action>
  <zorder>Next</zorder>
  <zorder>textEdit</zorder>
  <zorder>otvet1</zorder>
  <zorder>otvet2</zorder>
  <zorder>otvet3</zorder>
  <zorder>otvet4</zorder>
  <zorder>Vopros</zorder>
  <zorder>area</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
